### ISSUE TITLE
Is `__NO_MATH_INLINES` still necessary?

### ISSUE THREAD

**Issue by jpellegrini** [egallesio/STklos#532]
@egallesio -- in the beginning of `number.c`, there is this `define`:

```c
/* workaround for bad optimisations done in glibc2.1. Thanks to  Andreas Jaeger
 * <aj@suse.de> for it
 */
#ifndef __NO_MATH_INLINES
#  define __NO_MATH_INLINES
#endif
```
Is this still an issue? At least the time to run the test suite is the same without it.

**Comment by egallesio** [comment]
Hi @jpellegrini ,

Sorry for the long delay I take to answer these days. I'm actually fighting with different (non STklos) things and hope to be more operational soon. 

In any case, feel free to delete this workaround, which should not be necessary now.

**Comment by jpellegrini** [comment]
Ok @egallesio !
I hope everything is good with you. Tell me in private if I can be of any help.

**Comment by egallesio** [comment]
Yes, everything is alright. Just a lot of other things which take a lot of time. 
Thanks a lot for your help. @jpellegrini.

I have seen that you have fixed it in PR #533, so I close this issue.
